_posts/2021-08-04-cockpit-250.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Cockpit 250
+author: martin
```

Comment:
If you would use `pitti` it would also replace it with you full name and link to your aticles, as [here](https://cockpit-project.org/blog/pcp-grafana.html) for example.

Reply:
Ah, I already had that locally, just forgot to push. Thanks!